graphql-dgs-spring-boot-oss-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsAutoConfiguration.kt

```diff
@@ -154,8 +156,14 @@ open class DgsAutoConfiguration(
     }
 
     @Bean
-    open fun dgsDataLoaderProvider(applicationContext: ApplicationContext, dataloaderOptionProvider: DgsDataLoaderOptionsProvider): DgsDataLoaderProvider {
-        return DgsDataLoaderProvider(applicationContext, dataloaderOptionProvider)
+    @ConditionalOnMissingBean
+    open fun scheduledExecutorService(): ScheduledExecutorService {
+        return Executors.newSingleThreadScheduledExecutor()
```

Comment:
I think this will cause problems if someone is trying to autowire an `Executor` bean, since `ScheduledExecutorService` also implements `Executor`.

Reply:
Thanks, yeah I am changing it to add a quaifier for `dgsScheduledExecutorService`. Would that mitigate the issue you think?

Reply:
Yeah, a Qualifier would avoid the problem